x-pack/test/functional/apps/monitoring/_get_lifecycle_methods.js

```diff
@@ -14,6 +14,7 @@ export const getLifecycleMethods = (getService, getPageObjects) => {
   return {
     async setup(archive, { from, to, useSuperUser = false }) {
       _archive = archive;
+      await security.testUser.setRoles(['monitoring_user', 'kibana_admin']);
```

Comment:
can the `kibana_admin` role be removed or set to more fine grained feature control role ?

Reply:
`kibana_admin` is required to view monitoring data https://www.elastic.co/guide/en/kibana/current/monitoring-data.html.

I do get a "Permission Denied" message when trying to access the app without that role.